tests/integration/test_charm.py

```diff
@@ -171,10 +131,73 @@ async def test_scale_workload(
     kube_core_client.delete_namespaced_pod(name=leader_pod, namespace=ops_test.model_name)
     await ops_test.model.wait_for_idle(status="active")
 
-    unit_informations = json.loads(
-        (await ops_test.juju("show-unit", app.units[0].name, "--format", "json"))[1]
+    response = requests.get(
+        f"http://{await utils.get_mattermost_ip(ops_test, app)}:8065", timeout=5
     )
-    mattermost_ip = unit_informations[app.units[0].name]["address"]
-    response = requests.get(f"http://{mattermost_ip}:8065", timeout=5)
     assert response.status_code == 200
     assert "Mattermost" in response.text
+
+
+async def test_reminder_plugin(
+    ops_test: OpsTest,
+    app: Application,
+    test_entities,
+):
+    """
+    arrange: after charm is deployed and ready
+    act: enable the reminder plugin and execute a command to create a reminder
+    assert: the plugin should be enabled and the reminder should be produced
+    """
+
+    plugin_name = "com.github.scottleedavis.mattermost-plugin-remind"
+    cmd = (
+        "MMCTL_LOCAL_SOCKET_PATH=/tmp/mattermost.socket /mattermost/bin/mmctl --local"
+        f" plugin enable {plugin_name}"
+    )
+    await ops_test.juju("run", "--application", app.name, cmd)
+
+    cmd = (
+        "MMCTL_LOCAL_SOCKET_PATH=/tmp/mattermost.socket /mattermost/bin/mmctl --local"
+        " plugin list"
+    )
+    output = await ops_test.juju("run", "--application", app.name, cmd)
+    print(output)
+    for line in output[1].splitlines():
+        if line.startswith(plugin_name):
+            assert True, f"{plugin_name} is in enabled plugins."
+            break
+        if line == "Listing disabled plugins":
+            assert False, f"{plugin_name} is not in enabled plugins."
+            break
```

Comment:
Isn't it better to use a "found" boolean you set to true when you match
And then assert "found" ?

Reply:
It would be one more variable and I would have to change the assert message based on it.

```
    found = False
    for line in output[1].splitlines():
        if line.startswith(plugin_name):
            found = True
            break
        if line == "Listing disabled plugins":
            break
    assert found, f"{plugin_name} is {"" if found else "not"} in enabled plugins.
```

Is it better ?

Reply:
Fixed